code/__DEFINES/chat_box_defines.dm

```diff
@@ -6,3 +6,5 @@
 #define chat_box_notice(str) ("<div class='boxed_message notice_border'>" + str + "</div>")
 #define chat_box_healthscan(str) ("<div class='boxed_message notice_border left_align_text'>" + str + "</div>")
 #define chat_box_notice_thick(str) ("<div class='boxed_message notice_border thick_border'>" + str + "</div>")
+#define chat_box_ahelp(str) ("<div class='boxed_message red_border'>" + str + "</div>")
+#define chat_box_mhelp(str) ("<div class='boxed_message notice_border'>" + str + "</div>")
```

Comment:
```suggestion
#define chat_box_ahelp(str) ("<div class='boxed_message red_border'>" + str + "</div>")
#define chat_box_mhelp(str) ("<div class='boxed_message notice_border'>" + str + "</div>")
```
Why you ain't use chat_box_red and chat_box_notice?

Reply:
Because these are explicitly used for mhelp and ahelp alone. Defining them according to their usage, rather than their details, makes it easier to change their styling later.

Reply:
> Because these are explicitly used for mhelp and ahelp alone. Defining them according to their usage, rather than their details, makes it easier to change their styling later.

So if someone changes their style later on, they'll change
I don't see the point of just duplicating the macro.